waves/wallet/src/logger.rs

```diff
@@ -0,0 +1,16 @@
+use crate::storage::Storage;
+use anyhow::{Context, Result};
+
+pub fn try_init() -> Result<()> {
+    let local_storage = Storage::local_storage()?;
+
+    let log_level = local_storage
+        .get_item::<log::Level>("wallet_log")
+        .context("failed to get `wallet_log` log level")?;
+
```

Comment:
Will the user see an error message if `wallet_log` is not set in localStorage?

Reply:
Nope, by default the console will be quiet.

However, I don't think it is an _error_ to not set it. The logger is a developer / debugger setting so the ordinary user should not need to concern themselves with it.

Anyone interested in hacking on this thing can always read the documentation (README.md) to learn how to enable the logs.